src/main/common/huffman_table.c

```diff
@@ -28,262 +28,262 @@
-    { 10, 0x0D80 }, // 0xB8 0000110110
-    { 10, 0x0D40 }, // 0xB9 0000110101
-    { 10, 0x0D00 }, // 0xBA 0000110100
-    { 10, 0x0CC0 }, // 0xBB 0000110011
-    { 10, 0x0C80 }, // 0xBC 0000110010
-    { 10, 0x0C40 }, // 0xBD 0000110001
-    { 10, 0x0C00 }, // 0xBE 0000110000
-    { 10, 0x0BC0 }, // 0xBF 0000101111
-    { 10, 0x0B80 }, // 0xC0 0000101110
-    { 10, 0x0B40 }, // 0xC1 0000101101
-    { 10, 0x0B00 }, // 0xC2 0000101100
-    { 10, 0x0AC0 }, // 0xC3 0000101011
-    { 10, 0x0A80 }, // 0xC4 0000101010
-    { 10, 0x0A40 }, // 0xC5 0000101001
-    { 10, 0x0A00 }, // 0xC6 0000101000
-    { 10, 0x09C0 }, // 0xC7 0000100111
-    { 10, 0x0980 }, // 0xC8 0000100110
-    { 10, 0x0940 }, // 0xC9 0000100101
-    { 10, 0x0900 }, // 0xCA 0000100100
-    { 10, 0x08C0 }, // 0xCB 0000100011
-    { 10, 0x0880 }, // 0xCC 0000100010
-    { 10, 0x0840 }, // 0xCD 0000100001
-    { 10, 0x0800 }, // 0xCE 0000100000
-    { 10, 0x07C0 }, // 0xCF 0000011111
-    { 10, 0x0780 }, // 0xD0 0000011110
-    { 10, 0x0740 }, // 0xD1 0000011101
-    { 10, 0x0700 }, // 0xD2 0000011100
-    { 10, 0x06C0 }, // 0xD3 0000011011
-    { 10, 0x0680 }, // 0xD4 0000011010
-    { 11, 0x0320 }, // 0xD5 00000011001
-    { 10, 0x0640 }, // 0xD6 0000011001
-    { 10, 0x0600 }, // 0xD7 0000011000
-    { 10, 0x05C0 }, // 0xD8 0000010111
-    { 10, 0x0580 }, // 0xD9 0000010110
-    { 10, 0x0540 }, // 0xDA 0000010101
-    { 10, 0x0500 }, // 0xDB 0000010100
-    { 10, 0x04C0 }, // 0xDC 0000010011
-    { 11, 0x0300 }, // 0xDD 00000011000
-    { 10, 0x0480 }, // 0xDE 0000010010
-    { 10, 0x0440 }, // 0xDF 0000010001
-    {  9, 0x2900 }, // 0xE0 001010010
-    { 10, 0x0400 }, // 0xE1 0000010000
-    { 10, 0x03C0 }, // 0xE2 0000001111
-    { 11, 0x02E0 }, // 0xE3 00000010111
-    { 10, 0x0380 }, // 0xE4 0000001110
-    { 11, 0x02C0 }, // 0xE5 00000010110
-    { 11, 0x02A0 }, // 0xE6 00000010101
-    { 11, 0x0280 }, // 0xE7 00000010100
-    { 11, 0x0260 }, // 0xE8 00000010011
-    { 11, 0x0240 }, // 0xE9 00000010010
-    { 11, 0x0220 }, // 0xEA 00000010001
-    { 11, 0x0200 }, // 0xEB 00000010000
-    { 11, 0x01E0 }, // 0xEC 00000001111
-    { 11, 0x01C0 }, // 0xED 00000001110
-    { 11, 0x01A0 }, // 0xEE 00000001101
-    { 10, 0x0340 }, // 0xEF 0000001101
-    {  8, 0x3D00 }, // 0xF0 00111101
-    {  9, 0x2880 }, // 0xF1 001010001
-    { 11, 0x0180 }, // 0xF2 00000001100
-    { 11, 0x0160 }, // 0xF3 00000001011
-    { 11, 0x0140 }, // 0xF4 00000001010
-    { 11, 0x0120 }, // 0xF5 00000001001
-    { 11, 0x0100 }, // 0xF6 00000001000
-    { 11, 0x00E0 }, // 0xF7 00000000111
-    { 11, 0x00C0 }, // 0xF8 00000000110
-    { 12, 0x0010 }, // 0xF9 000000000001
-    { 11, 0x00A0 }, // 0xFA 00000000101
-    { 11, 0x0080 }, // 0xFB 00000000100
-    { 11, 0x0060 }, // 0xFC 00000000011
-    { 11, 0x0040 }, // 0xFD 00000000010
-    { 11, 0x0020 }, // 0xFE 00000000001
-    {  9, 0x2800 }, // 0xFF 001010000
-    { 12, 0x0000 }, // EOF  000000000000
+//             Len          Code        Char Bitcode
```

Comment:
It may be better to use macro, so that future refactoring is easier ... 
```
#define CODE(_len, _code) { .code_len = len, .code = code }
...
#undef CODE